docs/source/usage/examples.rst

```diff
@@ -51,10 +51,7 @@ Beam Distributions
    :maxdepth: 1
 
    examples/initialize_from_array/README.rst
-
-.. note::
-
-   We need to document the ``examples/distgen/`` examples here.
+   examples/distgen/README.rst
 
```

Comment:
I don't quite understand what determines the placement of line breaks here.  When I view this file under Preview, I see line breaks between some subsection lists, and not between others.  For example, here I see no line break associated with lines 53 and 54.